forge-gui/res/editions/Jumpstart Historic Horizons.txt

```diff
@@ -37,755 +37,380 @@ ScryfallCode=J21
+646 U Squirrel Sovereign @Ilse Gort
+647 R Squirrel Wrangler @Carl Critchlow
+648 C Striped Bears @Una Fricker
+649 R Sylvan Anthem @Franz Vohwinkel
+650 C Tajuru Pathwarden @Victor Adame Minguez
+652 U Tempered Sliver @Mitchell Malloy
+654 U Timeless Witness @Deruchenko Alexander
+655 U Tireless Provisioner @Josu Hernaiz
+659 C Trumpeting Herd @Lars Grant-West
+660 C Twin-Silk Spider @Ben Maier
+661 U Ulvenwald Mysteries @Greg Opalinski
+662 C Urban Daggertooth @Randy Vargas
+664 R Verdant Command @Mark Poole
+666 U Webweaver Changeling @Nicholas Gregory
+670 C Winding Way @Adam Paquette
+672 U Wren's Run Hydra @Filip Burburan
+677 U Arcbound Shikari @Campbell White
+678 U Arcus Acolyte @Volkan Baǵa
+680 C Breathless Knight @Yeong-Hao Han
+681 U Bred for the Hunt @Karl Kopinski
+682 C Captured by Lagacs @Andrew Mar
+683 R Chainer, Nightmare Adept @Steve Prescott
+684 C Chrome Courier @Hector Ortiz
+685 R Cloudshredder Sliver @Filip Burburan
+686 U Combine Chrysalis @Hector Ortiz
+687 C Drey Keeper @Kev Walker
+688 U Elusive Krasis @Wesley Burt
+689 U Etchings of the Chosen @Chuck Lukacs
+691 M The First Sliver @Svetlin Velinov
+693 C Goblin Anarchomancer @Joe Slucher
+694 U Good-Fortune Unicorn @Kee Lo
+695 U Graceful Restoration @Robbie Trevino
+697 U Ingenious Infiltrator @Jason Rainville
+701 U Lavabelly Sliver @Mark Behm
+702 U Lazotep Chancellor @Josu Hernaiz
+703 R Lonis, Cryptozoologist @Andrew Mar
+705 U Munitions Expert @Jesper Ejsing
+706 U Nimbus Swimmer @Howard Lyon
+707 R Priest of Fell Rites @Pauline Voss
+708 U Prophetic Titan @Slawomir Maniak
+709 U Rakdos Headliner @Ekaterina Burmak
+710 R Reap the Past @John Severin Brassell
+712 U Rotwidow Pack @Kev Walker
+713 U Ruination Rioter @Dmitry Burmak
+714 C Shambleshark @Wesley Burt
+718 U Soulherder @Seb McKinnon
+720 R Sterling Grove @Seb McKinnon
+721 C Storm God's Oracle @Pauline Voss
+722 R Sythis, Harvest's Hand @Ryan Yee
+723 C Terminal Agony @Lucas Graciano
+724 R Territorial Kavu @E. M. Gist
+725 U Thundering Djinn @Anastasia Ovchinnikova
+726 C Wavesifter @Nils Hamm
+728 U Ravenous Squirrel @Dan Murayama Scott
+730 U Fast // Furious @Deruchenko Alexander
+733 U Altar of the Goyf @Yeong-Hao Han
+734 C Amorphous Axe @Martina Pilcerova
+735 U Batterbone @Nicholas Gregory
+736 U Birthing Boughs @Mike Bierek
+738 C Bonded Construct @Craig J Spearing
+744 C Implement of Combustion @Enora Mercier
+748 U Monoskelion @Jason A. Engle
+749 C Myr Enforcer @Jim Murray
+750 C Myr Scrapling @Svetlin Velinov
+752 R Nettlecyst @Vincent Proce
+753 C Ornithopter of Paradise @Raoul Vitale
+755 U Sanctuary Raptor @Drew Baker
+758 U Treasure Keeper @James Paick
+759 C Universal Automaton @Ben Maier
+762 R Zabaz, the Glimmerwasp @Jason Felix
+763 C Ichor Wellspring @Steven Belledin
+770 C Khalni Garden @Ryan Pancoast
+776 R Sliver Hive @Igor Kieryluk
 
 [rebalanced]
 A438 U A-Dragon's Rage Channeler @Martina Fackova
 A529 C A-Unholy Heat @Kari Christensen
 
 [tokens]
```

Comment:
J21 doesn't have token images, so these shouldn't be defined there?

Reply:
Indeed I shouldn't. I'll fix it.